### PR TITLE
Add some sites

### PR DESCRIPTION
**Pull request by adisubru**

### CODE REVIEW

**Comment by tupaschoal** [comment]
Did you face any issue @adisubru , can I help you?

**Comment by adisubru** [comment]
I decided to add a few more sites, and thought i'd bunch all of them in one PR. Sorry for the trouble.

**Comment by adisubru** [comment]
added entries for MongoDB, Codeforces, CodeChef, WaniKani, AtCoder, Element, Zulip, Floatplane and Mangadex

File: _data/sites.json [hunk 1]
```diff
@@ -2842,6 +2881,16 @@
         ]
     },
 
+    {
+        "name": "Floatplane",
+        "url": "https://www.floatplane.com/legal/terms",
+        "difficulty": "hard",
+        "notes": "You need to contanct the Floatplane team through their <a hreaf=\"https://www.floatplane.com/support\">official website</a> to have your account deleted",
```
Comment (tupaschoal) [inline _data/sites.json:2888]:
```suggestion
        "url": "https://www.floatplane.com/support",
        "difficulty": "hard",
        "notes": "You need to contanct the Floatplane team through their official website to have your account deleted",
```

File: _data/sites.json [hunk 2]
```diff
@@ -8505,6 +8576,16 @@
         ]
     },
 
+    {
+        "name": "WaniKani",
+        "url": "https://www.wanikani.com/settings/danger_zone",
+        "difficulty": "easy",
+        "notes": "Visit the given link and click on \"Send deletion confirmation mail\". You will recieve a mail on your registered mail address and you can confirm the account deletion from there.",
```
Comment (tupaschoal) [inline _data/sites.json:8583]:
```suggestion
        "notes": "Visit the given link and click on \"Send deletion confirmation mail\". You will receive a mail on your registered mail address and you can confirm the account deletion from there.",
```

File: _data/sites.json [hunk 3]
```diff
@@ -5264,6 +5323,18 @@
         ]
     },
 
+    {
+        "name": "Mongo Atlas",
+        "url": "https://docs.atlas.mongodb.com/tutorial/delete-atlas-account/",
+        "difficulty": "hard",
+        "notes": "To delete your MongoDB account you first need to delete all active clusters, projects and organizations linked with your account (<a href=\"https://docs.atlas.mongodb.com/tutorial/delete-atlas-account/\">Instructions here</a>). Then you can send a mail to cloud-account-support@mongodb.com, with the subject line \"Request for Atlas Account Deletion\" from your regitered mail and delete your account",
```
Comment (tupaschoal) [inline _data/sites.json:5330]:
```suggestion
        "notes": "To delete your MongoDB account you first need to delete all active clusters, projects and organizations linked with your account. Then you can send a mail to cloud-account-support@mongodb.com, with the subject line \"Request for Atlas Account Deletion\" from your registered mail and delete your account",
```
I suggest removing the link from the text as it is already the link for the entry

**Comment by tupaschoal** [comment]
Thanks for all these entries!

**Pull request merged**